.eslintrc.js

```diff
@@ -9,15 +9,21 @@ module.exports = {
     'plugin:react/recommended',
     'plugin:prettier/recommended',
   ],
-  overrides: [],
+  overrides: [
+    {
+      files: ['**/__test__/**/*.[jt]s?(x)', '**/?(*.)+(spec |test).[jt]s?(x)'],
+      extends: ['plugin:testing-library/react'],
+    },
+  ],
   parserOptions: {
     ecmaVersion: 'latest',
     sourceType: 'module',
   },
-  plugins: ['react', 'prettier'],
+  plugins: ['react', 'prettier', 'testing-library'],
   rules: {
     'react/react-in-jsx-scope': 0,
     'react/jsx-props-no-spreading': 0,
     'react/jsx-filename-extension': 0,
+    'no-undef': 'off',
```

Comment:
eslint doesn't allow the use of undeclared variables. Since we are using `it`, `describe`, `expect` etc. it's giving error so had to add this rule

Reply:
> eslint doesn't allow the use of undeclared variables. Since we are using `it`, `describe`, `expect` etc. it's giving error so had to add this rule

I see